homeassistant/components/yalexs_ble/config_flow.py

```diff
@@ -166,6 +177,51 @@ async def async_step_integration_discovery_confirm(
             },
         )
 
+    async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
+        """Handle configuration by re-auth."""
+        self._reauth_entry = self.hass.config_entries.async_get_entry(
+            self.context["entry_id"]
+        )
+        return await self.async_step_reauth_validate()
+
+    async def async_step_reauth_validate(self, user_input=None):
```

Comment:
No type annotations? Maybe add the integration to strict typing if we're close?

Reply:
oops. That would have been nice. Will adjust and add strict typing

Reply:
https://github.com/home-assistant/core/pull/88086